### ISSUE TITLE
Data compilation of experiments

### ISSUE THREAD

**Issue by mlee1721** [AguaClara/humic_acid#42]

**Comment by mlee1721** [comment]
The plots show the ratio of the measured absorbance over the projected absorbance versus the measured turbidity (first graph) and concentration of humic acid (second graph).  Graphs show how our measured results compared against the expected results (accuracy).

![image](https://user-images.githubusercontent.com/43385922/55752274-901acf80-5a15-11e9-9e6f-aa5d8dd2587c.png)

![image](https://user-images.githubusercontent.com/43385922/55752286-98730a80-5a15-11e9-930f-d120d9a00451.png)

**Comment by monroews** [comment]
I think I need a caption for each graph describing what it is plotting.  What is the humic acid concentration in the first graph? Is the y axis the UV absorbance in both cases? I think these are the graphs that I requested, but now I'm having a hard time understanding what they mean!

IN the first graph wouldn't there be a plot for every different humic acid concentration?
In the second graph wouldn't there be a plot for every different turbidity?

**Comment by Carolynwamg** [comment]
The first Graph: 
1) X-Axis: The measured turbidity of all mixtures 
2) Y-Axis: Ratio of Measured absorbance over projected absorbance, this is to see whether our estimations are accurate or not. 

The second Graph: 
1)X-Axis: Humic Acid Concentrations at 5 mg/L, 10mg/L, 15mg/L, ( we’ll adjust it so it’s clearer, the current graph doesn’t show exactly these concentrations) 
2) Y-Axis: Ratio of Measured Absorbance of mixture and Projected absorbance, to test accuracy of projections

**Comment by monroews** [comment]
The first graph only has 6 data points. Is that the total number of mixtures that you tested?

**Comment by Carolynwamg** [comment]
No, we have 9 mixtures tested, but 3 of them were the results that maxed to 2 so we didn't plot it on.
